readthedocs/donate/templates/donate/promo_404.html

```diff
@@ -11,6 +11,7 @@
 {% endblock %}
 
 {% block content %}
+<div style="margin-left: auto; margin-right: auto; width: 900px; text-align: center">
```

Comment:
These rules don't need to be brought into core.css if they are specific to just this use case. Otherwise, a simple CSS include from a file in `donate/static` works. Inline styles should always be avoided as CSS can't be overhauled without sifting through templates on top of CSS.